WalletWasabi/Microservices/Binaries/osx64/Tor/tor

```diff
@@ -9,4 +9,4 @@ export DYLD_LIBRARY_PATH=.:$DYLD_LIBRARY_PATH
 # https://trac.torproject.org/projects/tor/ticket/10030
 cd "$(dirname "$0")"
 exec ./tor.real "$@"
-# Random Comment Dump to change the digest "je202kjnfoj2iuwkekjjreiejrc"
+# Random Comment Dump to change the digest "je202kjnfoj2iuwkekjjreiejrd"
```

Comment:
@kiminuo This is changed manually right? If yes don't you think we should mention it somewhere in the `UpgradeTorDaemons.md` document?

Reply:
Yes, we should have one. It is insanely absurd the amount of time we lose with this thing again and again.  Also, hashing only one file makes zero sense because we are also redistributing libraries.